### PR TITLE
Add nginx, systemd, AWS setup to deploy flow

### PR DESCRIPTION
**Pull request by tdooner**
This adds a number of things to the deployment scripts including some simple nginx + systemd configuration. The nginx needs some additional work, but I'm focusing on getting the credentials all provisioned at the moment.

See commit messages for details of each individual commit.

### CODE REVIEW

File: deploy/clientcomm.tf [hunk 1]
```diff
@@ -11,61 +11,77 @@ variable "ssh_public_key_path" {
...
 
+// TODO: This can probably be provisioned with terraform.
 variable "twilio_num" {
   description = ""
   default = "TODO ******TODO ******TODO *******"
 }
 
+// TODO: I think this is a constant, or at least derived from the hostname of
+// the deploy.
 variable "twilio_outbound_callback_url" {
```
Comment (tmaybe) [inline deploy/clientcomm.tf]: Yes, I believe you are correct on this TODO and the next; they are derived from the hostname of the deploy.

File: deploy/clientcomm.tf [hunk 2]
```diff
@@ -11,61 +11,77 @@ variable "ssh_public_key_path" {
...
 }
 
+// TODO: This will be determined from an RDS resource provisioned by terraform
 variable "database_host" {
   description = ""
   default = "TODO ******TODO ******TODO *******"
 }
 
+// TODO: Do we really need gmail integration here, or can we replace this
+// dependency with mailgun?
```
Comment (tmaybe) [inline deploy/clientcomm.tf]: We will eventually replace this dependency with mailgun or something like it.

File: deploy/clientcomm.tf [hunk 3]
```diff
@@ -11,61 +11,77 @@ variable "ssh_public_key_path" {
...
 }
 
+// TODO: This can probably be provisioned with terraform.
 variable "twilio_num" {
   description = ""
   default = "TODO ******TODO ******TODO *******"
 }
 
+// TODO: I think this is a constant, or at least derived from the hostname of
+// the deploy.
```
Comment (tmaybe) [inline deploy/clientcomm.tf]: Yes, I believe you are correct on this TODO and the next; they are derived from the hostname of the deploy.

File: deploy/clientcomm.tf (hunk 2 above)
Comment (tdooner) [inline deploy/clientcomm.tf]: Ah, great.

**Pull request merged**